model-providers/llama3-java/pom.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.quarkiverse.langchain4j</groupId>
+        <artifactId>quarkus-langchain4j-parent</artifactId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../../pom.xml</relativePath>
+    </parent>
+    <artifactId>quarkus-langchain4j-llama3-java-parent</artifactId>
+    <name>Quarkus LangChain4j - Llama 3 - Java - Parent</name>
+    <packaging>pom</packaging>
+
+    <properties>
+        <maven.compiler.release>21</maven.compiler.release>
```

Comment:
Really a Java 21 baseline?

Reply:
What's the alternative?

Reply:
17?

Reply:
Vector support on 17? Don't think so...

Reply:
Ahhhhh....You are right!

I was only looking at this from a JVM baseline - where Quarkus core baselines on 17. But you are right.